web-marketplace/src/legacy-pages/ProjectAccount/ProjectAccount.tsx

```diff
@@ -0,0 +1,180 @@
+
+import { AccountType } from 'generated/graphql';
+import { useAuth } from 'lib/auth/auth';
+
+import { ProjectPageFooter } from 'components/molecules';
+import Form from 'components/molecules/Form/Form';
+import { useZodForm } from 'components/molecules/Form/hook/useZodForm';
+import { AccountOption } from 'components/organisms/DashboardNavigation/DashboardNavigation.types';
+import { ProjectAccountSelector } from 'components/organisms/ProjectAccountSelector/ProjectAccountSelector';
+import { OnboardingFormTemplate } from 'components/templates/ProjectFormTemplate/OnboardingFormTemplate';
+import { useDaoOrganization } from 'hooks/useDaoOrganization';
+import { useQueryIsIssuer } from 'hooks/useQueryIsIssuer';
+
+import { useCreateProjectContext } from '../ProjectCreate/ProjectCreate';
+
+export const ProjectAccount = (): JSX.Element | null => {
+  const { _ } = useLingui();
+  const { projectId } = useParams();
+  const { activeAccount } = useAuth();
+  const dao = useDaoOrganization();
+  const {
+    projectCreatorAddress,
+    setProjectCreatorAddress,
+    setIsOrganizationAccount,
+    formRef,
+    isDraftRef,
+  } = useCreateProjectContext();
+
+  const personalAccount = useMemo(
+    (): AccountOption | null =>
+      activeAccount
+        ? {
+            name: activeAccount.name || '',
+            address: activeAccount.addr || '',
+            type: 'user' as const,
+            image:
+              activeAccount.image ||
+              getDefaultAvatar({
+                ...activeAccount,
+                type: activeAccount.type ?? AccountType.User,
+              }),
+            displayName: _(msg`Personal`),
+          }
+        : null,
+    [activeAccount, _],
+  );
+
+  const organizationAccount = useMemo(
+    (): AccountOption | null =>
+      dao
+        ? {
+            name: dao.organizationByDaoAddress?.name || '',
+            address: dao.address ?? '',
+            type: 'org' as const,
+            image: getDefaultAvatar({
+              type: AccountType.Organization,
+            }),
+            displayName: _(msg`Organization`),
+          }
+        : null,
+    [dao, _],
+  );
+
+  // Get the user's role and check if they have permission to create projects for the org
+  const organizationRole = dao?.assignmentsByDaoAddress?.nodes?.find(
+    node => node?.accountId === activeAccount?.id,
+  )?.roleName;
+
+  const shouldSkip =
+    !dao ||
+    !activeAccount ||
+    !personalAccount ||
+    !organizationAccount ||
+    !(organizationRole === 'owner' || organizationRole === 'admin');
+
+  const addressToCheck = projectCreatorAddress || activeAccount?.addr;
+
+  const { isIssuer, isLoadingIsIssuer } = useQueryIsIssuer({
+    address: addressToCheck,
+  });
+
+  const step = isIssuer ? 'choose-credit-class' : 'basic-info';
+
+  const { navigateNext } = useNavigateNext({
+    step,
+    projectId,
+  });
+
+  const form = useZodForm({
+    schema: z.object({}),
```

Comment:
what's the point of having a form with an empty schema?
we should store the selected account in the form values instead
if you replace your custom select field with our existing inputs/new/SelectTextField which handles react-hook-form values, this would work

Reply:
or let's not use react-hook-form at all here, although it would look cleaner